examples/v2_get_branches.rs

```diff
@@ -0,0 +1,32 @@
+//! Test the `GetBranches` endpoint.
+use anyhow::Context;
+use clap::AppSettings;
+use concordium_rust_sdk::v2;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+struct App {
+    #[structopt(
+        long = "node",
+        help = "GRPC interface of the node.",
+        default_value = "http://localhost:10001"
+    )]
+    endpoint: tonic::transport::Endpoint,
```

Comment:
Please use `endpoints::Endpoint` instead in these examples, where `endpoints` is imported from concordium-rust-sdk.

THis makes the example usable if somebody just imports concordium-rust-sdk, and not tonic directly.